playbook

```diff
@@ -0,0 +1,65 @@
+---
+- name: Install To Do app on server
+  hosts: managed
+  remote_user: ec2-user
+  vars_prompt:
+
+  - name: trello_key
+    prompt: Trello key
+    private: false
+
+  - name: trello_token
+    prompt: Trello token
+    private: false
+
+  tasks:
+  - name: Install Git
+    become: yes
+    ansible.builtin.yum:
+      name: git
+      state: present
+  
+  - name: Install Python
+    become: yes
+    ansible.builtin.yum:
+      name: python3
+      state: present
+
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSL https://install.python-poetry.org | python3
+      creates: ~/.local/bin/poetry
+
+  - name: Create To-Do App directory
+    ansible.builtin.shell:
+      cmd: mkdir -p ~/opt/todoapp
+      creates: ~/opt/todoapp
```

Comment:
This works nicely, but worth noting there is a built in option for this:
```suggestion
    ansible.builtin.file:
      path: ~/opt/todoapp
      state: directory
```

Reply:
Good suggestion, thank you!